packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -133,11 +133,11 @@ function isFocusable(element: Element) {
   }
 }
 
-export function getFirstFocusableDescendant(
+export function* getFocusableDescendants(
```

Comment:
Was the method not working properly before? I'm not against using generator functions, but this make things tad harder to grasp for fellow maintainers. If generators were widely used in the project thought I wouldn't object

Reply:
It was working properly, it's just that we were only able to get the first focusable descendant as opposed to all of them. I prefer to use a generator since we're traversing all the descendants of a given element which may not be very performant. I'd like `getFirstFocusableDescendant` to stop after it finds one focusable descendant.

Reply:
I think it's a bit overkill to think about performance in this case, since it's only used by e2e tests. Like we'll save couple milliseconds or something ? Not really worth it, IMO.

Also, if it's only used by e2e tests, it should be moved to the tests folder, and not be part of atomic source code itself.

Reply:
`getFocusableDescendants` is only used by e2e tests.

`getFirstFocusableDescendant` is only used by Atomic.

I did this to avoid code duplication.